src/main/java/mate/academy/spring/boot/controller/BookController.java

```diff
@@ -17,39 +20,39 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+@Tag(name = "Book management", description = "Endpoints for managing books")
+@RequiredArgsConstructor
 @RestController
-@RequestMapping("/books")
+@RequestMapping(value = "/api/books")
 public class BookController {
     private final BookService bookService;
 
-    @Autowired
-    public BookController(BookService bookService) {
-        this.bookService = bookService;
-    }
```

Comment:
why did you decide to remove the constructor injection? It's more preferable over field injection